### ISSUE TITLE
[Bug Report] activatable mixin 'activator' prop does nothing

### ISSUE THREAD

**Issue by dsanders11** [vuetifyjs/vuetify#7944]
### Versions and Environment
**Vuetify:** 2.0.0
**Vue:** 2.6.10
**Browsers:** Chrome 75.0.3770.142
**OS:** Mac OS 10.14.5

### Steps to reproduce
Reproduce case is simply a fork of the reproduce case from #5860, updated to Vuetify 2.0. That used to work for clicking, just not hovering, now it works for neither.

### Expected Behavior
The `activator` prop should be usable. This means event listeners should be bound dynamically, and unbound if the activator is changed.

### Actual Behavior
It's not useable.

### Reproduction Link
<a href="https://codepen.io/anon/pen/voGNog" target="_blank">https://codepen.io/anon/pen/voGNog</a>

### Other comments
It's clear to see from the code (https://github.com/vuetifyjs/vuetify/blob/master/packages/vuetify/src/mixins/activatable/index.ts) that this is simply not hooked up. There's no test coverage as well.

Despite 2 months ago me providing #7185 which fixed #5860, and commenting that this `activator` prop appeared hard broken in the `next` branch at the time, this still made it to 2.0 being broken.

**Comment by johnleider** [comment]
> Despite 2 months ago me providing #7185 which fixed #5860, and commenting that this activator prop appeared hard broken in the next branch at the time, this still made it to 2.0 being broken.

Not really necessary. At the time you created your fix, `master` was locked down with no changes at all. We asked for it to be rebased to `next` and you refused, hence nothing was merged/fixed.

**Comment by dsanders11** [comment]
> Not really necessary. At the time you created your fix, master was locked down with no changes at all. We asked for it to be rebased to next and you refused, hence nothing was merged/fixed.

There was a v1.5.16/v1.5.15 bugfix release from `master` after that date (those were released early June).

I declined to rebase onto `next` because it had changed too much (and was still in alpha) that it didn't make sense (for me) to try to fit it into code which had been redesigned and was still in flux. I didn't expect that to mean the bug wouldn't be fixed on v2.0.0.

I would have opened an issue, but that would have been a duplicate issue since that PR was already fixing an existing issue. What was the correct course of action to raise visibility, for future reference, @johnleider? Not trying to be a jerk or testy, it was just a frustrating experience as a contributor, so it would be nice if it was more clear what the right move was there.
